python/pyarrow/_csv.pyx

```diff
@@ -59,26 +59,26 @@ cdef class ReadOptions(_Weakrefable):
         This will determine multi-threading granularity as well as
         the size of individual record batches or table chunks.
         Minimum valid value for block size is 1
-    skip_rows: int, optional (default 0)
+    skip_rows : int, optional (default 0)
```

Comment:
Did those by hand, I wanted to check what issues the docstrings had and thus had to look at them anyway. At that point fixing them was a minor deal.

If it doesn't already exist I guess that an automation to fix them wouldn't be too hard to build, but given that @kszucs has a PR to prevent broken/missing docstrings from happening again I hope this is the only time we have to do this work.

Reply:
@amol- yes, https://github.com/apache/arrow/pull/7732 should be ready depending on which rules do we want to enable at first.